### DISCUSSION TITLE
support for claude anthropic please

### DISCUSSION THREAD

**Discussion by james007tia** [open-webui/open-webui#5775] [General]
# THIS IS ALREADY SUPPORTED VIA FUNCTIONS

https://openwebui.com/f/justinrahb/anthropic

**Comment by pavoltravnik** [comment]
Yeah, but web doesn't work for a whole day, so quite lame, there is single point of failure.

**Comment by beadon** [comment]
This does work, I have it running on my system actually.  What do you mean "web doesn't work for a whole day" ?  Were the API endpoints suddenly not available ?  I eeded to signup and join Anthropic's pas-by-the-query setup on the console here : https://console.anthropic.com/dashboard
